seqr/views/utils/export_utils.py

```diff
@@ -34,21 +40,21 @@ def export_table(filename_prefix, header, rows, file_format='tsv', titlecase_hea
         response = HttpResponse(content_type='text/tsv')
         response['Content-Disposition'] = 'attachment; filename="{}.tsv"'.format(filename_prefix)
         response.writelines(['\t'.join(header)+'\n'])
-        response.writelines(('\t'.join(map(unicode, row))+'\n' for row in rows))
+        response.writelines(('\t'.join(map(_to_str, row))+'\n' for row in rows))
```

Comment:
isn't `unicode` in python 2 basically the same as `str` in the future python 3? Because if so just swapping this out for future `str` should be fine. If we already weren't supporting unicode theres no reason to add support here

Reply:
Hmm, maybe you are right. I made it too complicated.